README.md

```diff
@@ -1,3 +1,34 @@
-# Extension Activity Monitor
+# Extension Activity Monitor [![Build Status](https://travis-ci.org/mozilla/extension-activity-monitor.svg?branch=master)](https://travis-ci.org/mozilla/extension-activity-monitor) [![Coverage Status](https://coveralls.io/repos/github/mozilla/extension-activity-monitor/badge.svg?branch=master)](https://coveralls.io/github/mozilla/extension-activity-monitor?branch=master)
 
-TBD
+This is a privileged extension for Firefox that monitors the activities of other installed extensions. The extension uses a privileged API called `activityLog` API to retrieve logs from other installed extensions.
```

Comment:
Nit, we could merge the two phrases I guess: "This is a privileged Firefox extensions that uses the `activityLog` API to monitor the activities of the other installed extensions."

Reply:
Thanks for the suggestion. Description updated.